api/metadata_processor_test.go

```diff
@@ -72,6 +75,7 @@ func (suite *MetadataProcessorSuite) TestDailyLesson() {
 		Part:           null.IntFrom(0),
 		Sources:        suite.someSources(),
 		Tags:           suite.someTags(),
+		Likutim:        suite.someLikutim(),
```

Comment:
testing likutim in `TestDdailyLesson` is enough. No real need in each and every flow.

Reply:
removed